environment_test.sh

```diff
@@ -5,3 +5,4 @@ export SMS_TEMPLATE_ID=31046c06-418a-49bf-86de-706b68415b47
 export NOTIFY_API_URL=https://api.notify.works
 export API_KEY=this-is-a-made-up-name_f869e87b-2b5d-4563-95f0-d8fe5bbd815c_80fee54f-01e7-4b48-8ab8-d8eae04be70e
 export EMAIL_REPLY_TO_ID=db8d1a9d-41ef-43cd-a04a-ed7d95214d95
+export SMS_SENDER_ID=e3b08b0e-323d-480b-8a3e-f972a75954d0
```

Comment:
What is this file for? It should only be for unit tests.

Reply:
Agreed. deleted, I just updated it from before